docs/general/ddl/data-types.rst

```diff
@@ -347,6 +351,66 @@ In these expressions, the desired time zone is specified as a string
 The scalar function :ref:`TIMEZONE <scalar-timezone>` (zone, timestamp) is
 equivalent to the SQL-conforming construct timestamp AT TIME ZONE zone.
 
+.. _time-data-type:
+
+time with time zone
```

Comment:
We should probably point out that this type cannot be used in CREATE TABLE statements.